.kitchen.yml

```
@@ -44,5 +49,9 @@ suites:
     run_list:
       - recipe[sudo]
       - recipe[user::data_bag]
-      - recipe[osl-app::app3]
       - recipe[app_test::app3]
+      - recipe[osl-app::app3]
+      - recipe[app_test::app3_postrun]
+    driver:
+      name: openstack
+      flavor_ref: 'm1.medium'
```

I've been having some issues with running out of RAM and would suggest you bump this up to large.
```suggestion
      flavor_ref: 'm1.large'
```